examples/log-errors/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <button id="err-btn" onclick="throwError();">Error!</button>
+    <button id="warn-btn" onclick="throwWarning();">Warning!</button>
```

Comment:
`throwWarning` can confuse.
[Warning](https://developer.mozilla.org/ru/docs/Web/API/Console/warn) is a well-known functionality.

It's better to use the `throwError1()` and `throwError2()` terms.